docs/en/CSMM/advanced-feature-guide-deadlydrops.md

```diff
@@ -21,32 +21,14 @@ zcoord :    (?<=Spawned supply crate at \(-?\d{1,4}\.\d, -?\d{1,4}\.\d, )(-?\d{1
 
 Note that we are avoiding the decimal points in the coordinates and only grabbing the integers, but when including the coordinates in the lookbehind for the zcoord variable we do need to account for the extra characters.
 
-We will use these coordinates to identify the entityId of the supply crate, which will use to create a unique name for the advanced claim in the next step. To get this, our command for this hook will be `w2l command "lce ${custom.xcoord} ${custom.zcoord} 5 SupplyCrate"`. This will look up any entities of type SupplyCrate within 5 blocks of the location of the airdrop, and will output a line including the coordinates again plus the supply crate ID.
+We will use these coordinates to create the claim, but we also need to make sure that the next airdrop doesn't try to create a claim with the same name. For this, we will use the `{{time}}` variable. This outputs the real-world time when the command is run, and it is unlikely that another supply drop will happen at exactly the same time on another day. We'll call our claim `DeadlyDrop{{time}}`.
```

Comment:
```suggestion
We will use these coordinates to create the claim, but we also need to make sure that the next airdrop doesn't try to create a claim with the same name. For this, we will use the `{{time}}` variable. This outputs the real-world time when the command is run, and it is unlikely that another supply drop will happen at exactly the same time on another day. We'll call our claim `DeadlyDrop\{\{time\}\}`.
```

Wasn't rendering properly for me, I think this'll fix it
![image](https://user-images.githubusercontent.com/22315101/127715811-28e1090a-ac85-4611-b4ad-da3113bb5c6b.png)

Reply:
Oh yeah definitely. I always forget whether I need escapes on those.